src/js/components/our-agents.js

```diff
@@ -0,0 +1,9 @@
+const tableBody = document.querySelector('.agents__body')
+const tableFooter = document.querySelector('.agents__footer')
+
+window.addEventListener('resize', setWidth)
```

Comment:
Probably since there is an adaptive approach was used (content width changed iteratively) there is no need to have a resize listener. Tracking resize could be replaced with the following - https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia

Reply:
Or at least add listener only when content size is really responsive.